OpenShift-Installation/eVDF-Openshift-Installation.rst

```diff
@@ -0,0 +1,592 @@
+         -f <format>         Certificate Format [pem|jks|p12]
+         -t <type>           Certificate Type [client|server|vsc|vrs]
+         -s <scpUrl>         The remote scp url. (eg. root@myhost://home/certs/)
+
+
+    .. Note:: The above command generates the client certificates for the "ose-admin" user and copies it to the /usr/local/ or any specified directory of the OSE node where Ansible is run. This certificate information is used by the nuage-openshift-monitor to securely communicate with the VSD.
+
+Using Git
+-------------
+
+You need to have Git installed on your Ansible machine. Perform the following tasks:
+
+1. Make sure https://github.com is reachable from your Ansible machine.
+
+2. Setup SSH and access the master and the minion nodes, using the **ssh** command.
+
+.. Note:: set-up passwordless **ssh** between Ansible node and cluster nodes.
+   
+3. Copy the nuage-ose-rhel-install-<version>.tar file shipped with the specific Nuage Release to a host machine where Ansible is run.
+
+4. Untar the above image
+
+   ::
+   
+       [root@ovs-10 ~]# tar -xvf nuage-ose-rhel-install.tar 
+       etcd_certificates.yml
+       nuage-infra-pod-config-daemonset.j2
+       nuage-openshift-ansible.diff
+       patch-nuage-openshift-ansible.sh
+
+      
+5. Run the patch-nuage-openshift-ansible.sh script to clone the ansible repo and set up Nuage changes.
+
+   .. Note:: The tag version may change based on the Nuage release you use.
+   
+   ::
+
+      [root@ansible-mc ~]# ./patch-nuage-openshift-ansible.sh 
+      Cloning into 'openshift-ansible'...
+      remote: Counting objects: 74439, done.
+      remote: Compressing objects: 100% (7/7), done.
+      remote: Total 74439 (delta 4), reused 2 (delta 2), pack-reused 74430
+      Receiving objects: 100% (74439/74439), 18.85 MiB | 3.60 MiB/s, done.
+      Resolving deltas: 100% (46284/46284), done.
+      Checking connectivity... done.
+      Note: checking out 'tags/openshift-ansible-3.7.0-0.116.0'.
+
+      You are in 'detached HEAD' state. You can look around, make experimental
+      changes and commit them, and you can discard any commits you make in this
+      state without impacting any branches by performing another checkout.
+
+      If you want to create a new branch to retain commits you create, you may
+      do so (now or later) by using -b with the checkout command again. Example:
+
+      git checkout -b <new-branch-name>
+
+      HEAD is now at cc47755... Automatic commit of package [openshift-ansible] release [3.7.0-0.116.0].
+      Successfully patched Nuage ansible changes into openshift-ansible
+      You may now use the openshift-ansible folder for your ansible installation
+
+
+Setup
+----------
+
+1. To prepare the OpenShift cluster for installation, follow the OpenShift Host Preparation guide.
+   
+   For Nuage releases prior to 5.2.1, go `here <https://docs.openshift.com/container-platform/3.5/install_config/install/host_preparation.html>`_.
```

Comment:
As eVDF is only supported with 5.3.2+ and OSP 3.7+, please remove the 5.2.1/3.5/3.6 sections and rename 5.2.2 to 5.3.2